publish.ps1

```diff
@@ -42,7 +42,7 @@ function do-prepareSource($build_server = "", $build_number = "")
 function do-build()
 {
     Write-Output "Building OpenRCT2..."
-    msbuild .\projects\openrct2.sln /p:Configuration=Release /p:Platform=Win32 /t:rebuild /v:minimal
+    msbuild .\projects\openrct2.sln /p:Configuration=$Env:CONFIGURATION /p:Platform=Win32 /t:rebuild /v:minimal
```

Comment:
This is ok as it fits nicely with AppVeyor's config spec, but I think it needs to have a default value if its not set (otherwise this will probably fail).